test/cqlpy/test_secondary_index.py

```diff
@@ -192,6 +192,15 @@ def test_create_index_if_not_exists2(cql, test_keyspace, cassandra_bug):
         with pytest.raises(InvalidRequest, match="already exists"):
             cql.execute(f"CREATE INDEX IF NOT EXISTS {index_name} ON {table}(v2)")
 
+# Verify that oversized index names are cleanly rejected  as InvalidRequest
+# Reproduces issue #20755
+def test_create_index_oversized_name(cql, test_keyspace, cassandra_bug):
+    with new_test_table(cql, test_keyspace, 'p int primary key, v int') as table:
+        index_name = 'x'*500
+        # Obviously can't create an index with the oversized name:
```

Comment:
nitpick: this comment doesn't help - it just says what the line below it does (and also repeats the same explanation as the comment above the test).

Reply:
removed